chromium_src/content/browser/permissions/permission_controller_impl.cc

```diff
@@ -5,9 +5,15 @@
 
 #include "content/browser/permissions/permission_controller_impl.h"
 
+#include "brave/components/brave_wallet/common/buildflags/buildflags.h"
 #include "content/browser/permissions/permission_util.h"
 #include "third_party/blink/public/common/permissions/permission_utils.h"
 
+#define BRAVE_WALLET_NUM_CASES         \
+  case PermissionType::BRAVE_ETHEREUM: \
+  case PermissionType::BRAVE_SOLANA:   \
+  case PermissionType::BRAVE_CARDANO:
+
 #define NUM                                             \
```

Comment:
Might be reverted

Reply:
Fixed in https://github.com/brave/brave-core/pull/32565/commits for this commit title.